### PR TITLE
Auto increment fix

### PR DESCRIPTION
**Pull request by jacobpenny**
The hard-coded values in the ExternalLinks table assumed that the MYSQL variables `auto_increment_increment` and `auto_increment_offset` are set to 1 and 0 respectively. This often isn't the case in multi-master clusters.

The patch is done in the same way as:
https://github.com/aces/Loris/blob/17.1-dev/SQL/Release_patches/15.10_To_16.0_upgrade.sql#L11
I'm assuming that the ExternalLinks table is static and not updated/appended to after install.

### CODE REVIEW

**Comment by xlecours** [comment]
Adding discussion required: 
@jacobpenny Is this for fresh install only? if yes, a patch might not be required
If not, can project put anything in that table? Should we try to keep the table content instead of dropping it?

**Comment by jacobpenny** [comment]
I'm not 100% sure on any of your questions so discussion is definitely required, good call.
It isn't likely to affect existing projects (they would be obviously broken), so maybe a patch isn't required. Judging from the 15.10_To_16.0 patch, I don't think users can put anything in the table. Will confirm at next PR meeting.

File: SQL/Archive/17.1/2017-01-24-UpdateLinkTypeReference.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS ExternalLinks;
+CREATE TABLE ExternalLinks (
```
Comment (MounaSafiHarab) [inline SQL/Archive/17.1/2017-01-24-UpdateLinkTypeReference.sql]: no need for the patch

**Comment by jacobpenny** [comment]
@xlecours Removed patch as discussed

**Comment by davidblader** [comment]
this is good :+1:

**Pull request merged**